eventually/team/models.py

```diff
@@ -181,13 +186,14 @@ def update(self, owner=None, members=None, name=None, description=None, image=No
 
         :param image: team's image
         :type image: str
-
         :return: None
         """
         if owner:
             self.owner = owner
-        if members:
-            self.members = members
+        if members_add:
```

Comment:
what if we would create additional functions `add_users` and `remove_users` like the model's methods?
what do you think?